qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/project/QuteProjectRegistry.java

```diff
@@ -411,14 +401,13 @@ public JavaMemberInfo findMember(ResolvedJavaTypeInfo resolvedType, String prope
 	 *
 	 * @return the member field retrieved by the given property and null otherwise.
 	 */
-	protected static JavaFieldInfo findField(ResolvedJavaTypeInfo resolvedType, String fieldName) {
-		List<JavaFieldInfo> fields = resolvedType.getFields();
-		if (fields == null || fields.isEmpty() || isEmpty(fieldName)) {
-			return null;
-		}
-		for (JavaFieldInfo field : fields) {
-			if (fieldName.equals(field.getName())) {
-				return field;
+	protected static JavaFieldInfo findField(ResolvedJavaTypeInfo baseType, String fieldName) {
+		List<JavaFieldInfo> fields = baseType.getFields();
+		if (!(fields == null || fields.isEmpty() || isEmpty(fieldName))) {

```

Comment:
previous guard condition is easier to read/understand

Reply:
fixed (like before)